package.json

```diff
@@ -12,19 +12,19 @@
     "clean": "pnpm turbo clean && pnpm storybook:clean && rimraf node_modules",
     "test": "pnpm turbo test",
     "test:ci": "pnpm turbo test:ci",
-    "storybook": "storybook dev -p ${STORYBOOK_PORT:-6006} -c storybook",
+    "storybook": "pnpm -F storybook storybook",
```

Comment:
Whilst I understand how this works, it does look a bit weird. Perhaps worth making a `pkg:storybook` like we did with `pkg:aio`?